api/receiver/receive_results.go

```diff
@@ -45,14 +45,14 @@ func HandleResultsUpload(a AppEngineAPI, w http.ResponseWriter, r *http.Request)
 		uploader = username
 	}
 
-	// Most form methods (e.g. PostFormValue, FormFile) will call
+	// Most form methods (e.g. FormValue, FormFile) will call
```

Comment:
Drive-by: PostFormValue requires the param to be in the POST body, but FormValue lets it be in the URL params _or_ body, with precendence for the body.

This can be quite useful when you're typing a CURL from the command-line.

Reply:
I'm slightly hesitant about this change. I usually prefer to have all params either in the body (for POST) or in the URL (for GET). My personal experience is that mixing them is error-prone. For local testing of POST, I often use [httpie](https://httpie.org/) instead of curl for cleaner syntax. But if you think this helps your development (we are a small team, after all), then please feel free to go ahead, but remember to add a comment regarding the precedence.

Reply:
Added a comment. I'm not proposing we mix them for real callers; just that supporting both is better for ad-hoc debugging work.